app/controllers/api/v1/puppetclasses_controller.rb

```diff
@@ -16,8 +16,8 @@ def index
           authorized(:view_puppetclasses).
           search_for(*search_options).paginate(paginate_options).
           select([:name, :id]).
-          includes(:lookup_keys)
-        render :json => Puppetclass.classes2hash(values.all)
+          includes(:lookup_keys).to_a
```

Comment:
I get it but I was wondering if we can avoid eager loading this on the API and wait until it's rendered?